### PR TITLE
components: updated custom fields components.

### PR DESCRIPTION
**Pull request by alejandromumo**
closes https://github.com/inveniosoftware/invenio-rdm-records/issues/1232

### CODE REVIEW

**Comment by alejandromumo** [comment]
Screenshot at Mar 08 16-48-10
Screenshot at Mar 08 16-48-01

**Review by jrcastro2 — COMMENTED** [review]
LGTM! Just a couple of comments.
When doing changes in the UI, it would be nice if we add some screenshots of the changes to visualize them easily.

**Comment by alejandromumo** [comment]
> LGTM! Just a couple of comments. When doing changes in the UI, it would be nice if we add some screenshots of the changes to visualize them easily.

Thanks for the comments!

The screenshots are attached to the [issue](https://github.com/inveniosoftware/invenio-rdm-records/issues/1232). 
I had them in this PR but then it made sense to move them there since the mockups are also there (easier to compare "side by side"

**Comment by jrcastro2** [comment]
> > LGTM! Just a couple of comments. When doing changes in the UI, it would be nice if we add some screenshots of the changes to visualize them easily.
> 
> Thanks for the comments!
> 
> The screenshots are attached to the [issue](https://github.com/inveniosoftware/invenio-rdm-records/issues/1232). I had them in this PR but then it made sense to move them there since the mockups are also there (easier to compare "side by side"

Ah, right! Thanks for the screenshots. I didn't scroll down and missed them, maybe it's worth mentioning in the PRs that the screenshots are available in the issue ;)

File: src/lib/components/Thesis.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    return (
+      <>
+        {description && (
+          <>
+            <FieldLabel htmlFor={fieldPath} icon={icon} label={description} />
+            <Divider fitted />
+          </>
+        )}
+        <Grid padded>
+          <Grid.Column width="12">
```
[the remark this answers is not in the document]
Comment (kpsherva) [reply src/lib/components/Thesis.js:25]: true, can be fixes in a new PR potentially

[the remark this answers is not in the document]
Comment (alejandromumo) [reply src/lib/components/Thesis.js]: Fixed :)

**Pull request merged**